lua/damagelogs/shared/lang/brazilian_portuguese.lua

```diff
@@ -234,7 +234,6 @@ DamagelogLang.portugues = {
     ViewChat = "Ver chat",
     UpdateNotifications = "Ativar notificação de atualização",
     Reported = "Reportado",
-    Victim = "Vítima",
```

Comment:
This is used in a few places
E.g. https://github.com/BadgerCode/tttdamagelogs/blob/master/lua/damagelogs/client/chat.lua#L66

Reply:
This is already in use on line 74 and I think this is redundant.

Reply:
Repeated string translations seem to use the first occurrence of the

Reply:
Ahh I see. Cheers!